packages/domain-picker/src/hooks/use-on-mount.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
@razvanpapadopol suggested in a call to move these generic helpers to a package. Did you suggest `@automattic/analyitcs` package specifically?

Reply:
There are packages of general purpose hooks packages. We should consider using one in our apps. Maybe a good post on codep2.

I know of at least a few places we implement hooks that could be shared more easily in a package.